tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.settings.app import Settings
+from marionette_driver import By
+
+
+class TestSettingsRTL(GaiaImageCompareTestCase):
+    _page_locator = (By.ID, 'root')
+
+    def setUp(self):
```

Comment:
Actually we need to run GaiaImageCompareTestCase.setUp(self), and I'd prefer to run it in the setUp() method since it's not part of the test.